WebService/v1/Models/SimulationApiModel/DeviceModelApiModelOverride.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Models;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Models.SimulationApiModel
+{
+    // SEE: <DeviceModelApiModel> for the original fields being overridden
```

Comment:
this class seems odd to me - why wouldn't you subclass the devicemodelapimodel class if that's what you're actually doing?

Reply:
the original class has a lot of properties that cannot be overridden and would show up in here even if they cannot be overridden in the business logic. Also all the initialization is different, this class properties are all null by default. There's also a coupling problem, where changes to one model would affect the other, with a higher risk of breaking the API in case of changes. A separate class gives in my opinion better control on the features provided (i.e. what can be overridden).